integration/src/main/java/scripts/testscripts/SnapshotDatasetOperations.java

```diff
@@ -132,7 +133,7 @@ private void previewUserJourney(CatalogClient client) throws ApiException {
     @SuppressWarnings("unchecked")
     Map<String, String> row = (Map<String, String>) sampleTable.getRows().get(0);
 
-    assertThat(row, hasEntry(is("sample_id"), is("sample5")));
+    assertThat(row, hasEntry(is("sample_id"), is(notNullValue())));
```

Comment:
Can you directly assert against `notNullValue()`?
```suggestion
    assertThat(row, hasEntry(is("sample_id"), notNullValue()));
```